tests/Integration/Behaviour/Features/Scenario/Product/Combination/Shop/delete_multishop_combination.feature

```diff
@@ -0,0 +1,40 @@
+# ./vendor/bin/behat -c tests/Integration/Behaviour/behat.yml -s product --tags delete-multi-shop-combination
+@restore-products-before-feature
+@clear-cache-before-feature
+@restore-shops-after-feature
+@clear-cache-after-feature
+@product-combination
+@delete-combination
+@product-multi-shop
+@delete-multi-shop-combination
+Feature: Delete combination from Back Office (BO) in multiple shops
+  As a BO user
+  I need to be able to delete product combinations from BO in multiple shops
+
+  Background:
```

Comment:
About these tests here are a few scenarios I can picture:
- associate products on multiple shops, delete for one specific shop, the combination is not deleted and is still associated to the other shops
- delete all the associations shops by shop, the combination should be deleted completely at the same time of the last association

Reply:
You will also need to add some tests regarding the combination generation amongst which:
- create product with combinations associated to one shop, associate the product to a shop, all the existing combinations are associated to the shop by default
- now generate some combinations for one of the two shops, the new combination are only associated to one shop
- finally generate same combinations for another shop, the previous combinations are still here and still the same but the associations have been added (no duplicate combinations were created in any shop)